pages/linux/goaccess.md

```diff
@@ -0,0 +1,20 @@
+# goaccess
+
+> GoAccess is an open source real-time web log analyzer.
+> More information: <https://goaccess.io>.
+
+- Run `goaccess` in interactive mode:
+
+`goaccess {{path/to/file}} [{{path/to/file}}]`
+
+- Use a specific log-format (or pre-defined formats like **combined**):
```

Comment:
I don't think any of the clients currently support bold tags, so probably best to use quotes.
```suggestion
- Use a specific log-format (or pre-defined formats like "combined"):
```